sub/source/docs/failsafe-landing-page.rst

```diff
@@ -0,0 +1,24 @@
+.. _failsafe-landing-page:
+
+========
+Failsafe
+========
+
+Sub has a number of failsafe mechanisms to ease vehicle recovery/prevent wandering in the event that vehicle control is lost. The main failsafe topics are listed below.
+
+.. note:: Once a Radio Failsafe, Battery Failsafe, GCS Failsafe or Terrain Data Loss Failsafe is entered and the failsafe action taken involves a mode change to the vehicle, it will remain in that mode until the pilot changes the mode directly.
```

Comment:
Terrain Data Loss isn't one of the listed ones. I'm unsure whether Sub supports it, but if not it should be removed from here.